executor/show.go

```diff
@@ -729,11 +735,26 @@ func (e *ShowExec) fetchShowCreateTable() error {
 	if len(tb.Meta().Comment) > 0 {
 		fmt.Fprintf(&buf, " COMMENT='%s'", format.OutputFormat(tb.Meta().Comment))
 	}
-
 	e.appendRow([]interface{}{tb.Meta().Name.O, buf.String()})
 	return nil
 }
 
+func (e *ShowExec) fetchShowCreateTable4View(tb *model.TableInfo, buf *bytes.Buffer) {
```

Comment:
As @eurekaka mentioned in the earlier code reviews. MySQL behaves different between normal table and view on `show create table`. For a normal table, the output has two columns, namely `Table` and `Create Table`:
```sql
MySQL(root@localhost:test) > create table t(a bigint, b bigint);
Query OK, 0 rows affected (0.11 sec)

MySQL(root@localhost:test) > show create table t;
+-------+----------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table                                                                                                                                       |
+-------+----------------------------------------------------------------------------------------------------------------------------------------------------+
| t     | CREATE TABLE `t` (
  `a` bigint(20) DEFAULT NULL,
  `b` bigint(20) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
+-------+----------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.01 sec)
```

But for View, the output contains 4 columns, namely `View`, `Create View`, `character_set_client` and `collation_connection`:
```sql
MySQL(root@localhost:test) > create view v as select * from t;
Query OK, 0 rows affected (0.11 sec)

MySQL(root@localhost:test) > show create table v;
+------+---------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+
| View | Create View                                                                                                                                             | character_set_client | collation_connection |
+------+---------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+
| v    | CREATE ALGORITHM=UNDEFINED DEFINER=`skip-grants user`@`skip-grants host` SQL SECURITY DEFINER VIEW `v` AS select `t`.`a` AS `a`,`t`.`b` AS `b` from `t` | utf8mb4              | utf8mb4_0900_ai_ci   |
+------+---------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+
1 row in set (0.00 sec)
```

Reply:
Done